public/components/datasources/components/manage/data_connection.tsx

```diff
@@ -368,6 +390,8 @@ export const DataConnection = (props: { dataSource: string }) => {
         return <S3DatasourceOverview />;
       case 'PROMETHEUS':
         return <PrometheusDatasourceOverview />;
+      default:
+        return null;
```

Comment:
I know we had this conversation before for the datasource type. However, I still think we need to setup a type for lake formation.

Reply:
As discussed, since lakeformation connection is a subtype for S3 we will not add a new data source type